index.html

```diff
@@ -4344,7 +4310,40 @@ <h2>
 
     <section>
         <h2>
-Immutability
+Persistence of DIDs
+        </h2>
+
+        <p>
+Although a <a>DID</a> is able to function as a Uniform Resource Name (URN)—a
+"persistent, location-independent resource identifier"as defined by [[RFC8141]]—
+this is not a requirement of all DIDs. Rather the persistence of a <a>DID</a>,
+is determined by the <a>DID controller</a>.
+
+If a <a>DID controller</a> intends for a <a>DID</a> to function as a URN, there
+are two important security considerations:
+        </p>
+        <ol start="1">
+          <li>
+The <a>DID controller</a> should choose a <a>DID method</a> that supports their
+requirements for persistence. <a>DID controllers</a> are advised to refer to the
+Decentralized Characteristics Rubric [[?DID-RUBRIC]] to decide which
+<a>DID method</a> best meets their persistence requirements.
+          </li>
+          <li>
+The <a>DID controller</a> should publish its operational policies so relying
+parties can determine the degree to which they can rely on the persistence of a
+<a>DID</a> controlled by that <a>DID controller</a>. In the absence of such
+policies, relying parties should not make any assumption about whether a
+<a>DID</a> is a persistent identifier for the same <a>DID subject</a>. This is
+particularly important from a security standpoint if the <a>DID</a> is used for
+authentication or authorization purposes.
+          </li>
+        </ol>
```

Comment:
No need for an ordered/numbered list here; the two points are independent
of each other. Also a couple of minor text/punctuation tweaks.
```suggestion
        <ul>
          <li>
The <a>DID controller</a> should choose a <a>DID method</a> that supports
their requirements for persistence. <a>DID controllers</a> are advised to
refer to the Decentralized Characteristics Rubric [[?DID-RUBRIC]] to decide
which <a>DID method</a> best meets their persistence requirements.
          </li>
          <li>
The <a>DID controller</a> should publish its operational policies so that
relying parties can determine the degree to which they can rely on the
persistence of a <a>DID</a> controlled by that <a>DID controller</a>. In
the absence of such policies, relying parties should not make any
assumption about whether a <a>DID</a> is a persistent identifier for the
<a>DID subject</a>. This is particularly important from a security
standpoint if the <a>DID</a> is used for authentication or authorization
purposes.
          </li>
        </ul>
```